### PR TITLE
Fix CPYTOIMPF for different default

### PR DESCRIPTION
**Pull request by dariocs**
### Changes

Description of change here.

### Checklist

* [x] have tested my change
* [x] updated relevant documentation
* [x] Remove any/all `console.log`s I added
* [x] eslint is not complaining
* [x] have added myself to the contributors' list in the README

### CODE REVIEW

**Comment by worksofliam** [comment]
@dariocs Is this PR finished?

**Comment by dariocs** [comment]
Yes , thanks

**Comment by worksofliam** [comment]
@dariocs Would you mind updating the text of the change in your PR and also complex the checklist?

Thanks!

**Comment by dariocs** [comment]
@worksofliam 
Hi Liam , 
I have done , I'm not so smart with git .. I'm beginner ... 
Thanks

File: src/api/IBMiContent.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = class IBMiContent {
       try {
         //If this command fails we need to try again after we delete the temp remote
         await this.ibmi.remoteCommand(
-          `CPYTOSTMF FROMMBR('${path}') TOSTMF('${tempRmt}') STMFOPT(*REPLACE) STMFCCSID(1208)`, '.'
+          `CPYTOSTMF FROMMBR('${path}') TOSTMF('${tempRmt}') STMFOPT(*REPLACE) STMFCCSID(1208) FLDDLM(',') DECPNT(*PERIOD)`, '.'
```
Comment (worksofliam) [inline src/api/IBMiContent.js:66]: Sadly this change is invalid.

`FLDDLM` and `DECPNT` are not valid commands on [CPYTOSTMF](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/cl/cpytostmf.htm). These will need to be removed before I can merge this commit.

File: src/api/IBMiContent.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ module.exports = class IBMiContent {
         ') ' +
         "TOSTMF('" +
         tempRmt +
-        "') MBROPT(*REPLACE) STMFCCSID(1208) RCDDLM(*CRLF) DTAFMT(*DLM) RMVBLANK(*TRAILING) ADDCOLNAM(*SQL)",
+        "') MBROPT(*REPLACE) STMFCCSID(1208) RCDDLM(*CRLF) DTAFMT(*DLM) RMVBLANK(*TRAILING) ADDCOLNAM(*SQL) FLDDLM(',') DECPNT(*PERIOD) ",
```
Comment (worksofliam) [inline src/api/IBMiContent.js:142]: Looks good to me!

**Review by worksofliam — CHANGES_REQUESTED** [review]
Please see my comments and make changes where necessary. When you are ready, please request a review again 😄

File: src/api/IBMiContent.js (hunk 1 above)
Comment (worksofliam) [reply src/api/IBMiContent.js:66]:
```
PD0043: Keyword FLDDLM not valid for this command.
CPD0043: Keyword DECPNT not valid for this command.
CPF0006: Errors occurred in command.
```

**Review by worksofliam — APPROVED** [review]

**Comment by worksofliam** [comment]
@dariocs I have retested and it's all looking good.

Thank you for your contribution!!

**Pull request merged**